AppOne/AppOne/HomeVC.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  HomeVC.swift
+//  AppOne
+//
+//  Created by Asapu, Ajaikumar on 19.11.22.
+//
+
+import UIKit
+
+class HomeVC: UIViewController {
```

Comment:
class should be final if we dont plan to subclass

Reply:
@JoeHoHH made it to final [here](https://github.com/JoeHoHH/ajayTheLead/pull/1/commits/82b81a4276343fca98befc7014edae74c8e77fa8)